truss/cli/cli.py

```diff
@@ -1220,14 +1245,17 @@ def push(
 
     """
     if not remote:
-        remote = inquire_remote_name(RemoteFactory.get_available_config_names())
+        remote = remote_cli.inquire_remote_name()
+
+    if not include_git_info and remote_cli.check_is_interactive():
```

Comment:
Should the `include_git_info` argument type be optional and this check be `include_git_info is None` instead of `not include_git_info`? The user could pass in `False` right?